cmd/main.go

```diff
@@ -25,6 +27,13 @@ func main() {
 		return
 	}
 
+	sigs := make(chan os.Signal, 1)
+	signal.Notify(sigs, syscall.SIGTERM)
+	go func() {
+		<-sigs
+		os.Exit(1)
```

Comment:
nit - print a message indicating we received an unexpected terminate signal

```suggestion
		fmt.Println("Error: received unexpected terminate signal")
		os.Exit(1)
```

Reply:
Updated. Thanks!